library/Zend/Tool/Project/Profile/Resource/Container.php

```diff
@@ -383,7 +383,7 @@ public function valid()
      */
     public function hasChildren()
     {
-        return (count($this->_subResources > 0)) ? true : false;
+        return (count($this->_subResources) > 0) ? true : false;
```

Comment:
Probably a type-o, but since it would have been counting `true` or `false`, behavior doesn't change.